app/js/timeagoLocale.js

```diff
@@ -1,45 +1,46 @@
 // @flow
+import * as langs from 'timeago.js/lib/lang';
 
 const timeagoLocale = {
-  get ar() { return require('timeago.js/locales/ar'); },
```

Comment:
This was doing dynamic requires so the user only has to load the locale they're using. By doing a static `import` at the top, every user will load every locale. Did you intend to change that?

Reply:
yeah. the new version of timeago.js changed things, and `require` simply doesn't work anymore, this is the cleanest way. yes there is a tiny overhead with it, but in the grand scheme of things it's not much.